packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -115,7 +115,7 @@ export default {
 
         showLegacyBanner: {
             type: Boolean,
-            default: false
+            default: null
         },
```

Comment:
For my understanding - why is the default `null` rather than `false`

Reply:
So I can check if the prop has been passed in `return this.showLegacyBanner === null ?`

Reply:
and it wouldn't work if this was `false` instead? I'm happy to approve I just want to make sure I understand

Reply:
If I set it to `false` and then pass in `false` from the consuming app, Vue doesn't know that there has been a change and will always use the prop value rather than check the tenant config. This would mean that every consuming app would need to pass in the correct value for each tenant every time.

Reply:
Ahhh cool, I understand now thank you 👍